src/CouchDB.Driver/Settings/CouchSettings.cs

```diff
@@ -29,6 +30,14 @@ public interface ICouchConfiguration
         /// <returns>The current settings</returns>
         ICouchConfiguration UseCookieAuthentication(string username, string password, int cookieDuration = 10);
 
+        /// <summary>
+        /// Enables proxy authentication. 
+        /// </summary>
+        /// <param name="username">Server username.</param>
+        /// <param name="roles">Server roles.</param>
```

Comment:
Token param missing

Reply:
Ok. (I have added `token` later and forgot to update comments.)